lib/src/home/widgets/article_card.dart

```diff
@@ -0,0 +1,51 @@
+import 'package:flutter/material.dart';
+import 'package:tech_tuesday/core/models/article_model.dart';
+import 'package:tech_tuesday/core/utils/constants.dart';
+import 'package:intl/intl.dart';
+import 'package:tech_tuesday/core/utils/routes.dart';
+
+class ArticleCard extends StatelessWidget {
+  final Article article;
+  const ArticleCard({Key? key, required this.article}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return GestureDetector(
+      onTap: () {
+        Navigator.of(context).pushNamed(
+          Routes.articleView,
+          arguments: article,
+        );
+      },
+      child: Container(
+        padding: const EdgeInsets.symmetric(
+          horizontal: 32,
+          vertical: 16,
+        ),
+        child: Column(
+          children: [
+            Image.network(article.imageUrl),
```

Comment:
the Image.network is supposed to fetch the image from the internet so in this instance are you fetching the image/s from article